libsolidity/analysis/ViewPureChecker.cpp

```diff
@@ -41,7 +41,6 @@ class AssemblyViewPureChecker: public boost::static_visitor<void>
 		m_dialect(_dialect),
 		m_reportMutability(_reportMutability) {}
 
-	void operator()(yul::Label const&) { }
 	void operator()(yul::Instruction const& _instruction)
```

Comment:
This should go.

Reply:
Done in PR https://github.com/ethereum/solidity/pull/7449